sidebars.js

```diff
@@ -8,154 +8,179 @@
 module.exports = {
   "docs": [
     {
-      "Welcome": [
+      "Get started": [
       "overview",
-      "saas-vs-self-hosted",
+      "self-hosted/install/preparation",
       {
         "type": "category",
-        "label": "Getting Started",
+        "label": "Installing Okteto",
         "items": [
-          "getting-started",
```

Comment:
we are missing these two. this is our current getting started guide. it should go right after the SH installation